frontend/app/runner/selectvenue/page.tsx

```diff
@@ -0,0 +1,63 @@
+"use client";
+
+import { Button } from "@/components/ui/button"
+import { BackButton } from "@/components/backbutton"
+import { useRouter } from "next/navigation";
+import { useEffect, useState } from "react";
+
+export default function Home() {
+  const API_URL = process.env.NEXT_PUBLIC_API_URL;
+  const router = useRouter();
+
+  const [venues, setVenues] = useState([]);
```

Comment:
Make sure to use Venue type from types folder.
```suggestion
  const [venues, setVenues] = useState<Venue[]>([]);
```